lib/maker.js

```diff
@@ -0,0 +1,108 @@
+'use strict';
+
+var _ = require('lodash');
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: ['_' is defined but never used.](https://www.codacy.com/app/jasoncalabrese/cgm-remote-monitor/file/758176594/issues/source?bid=2449627&fileBranchId=2449700#l3)